pkg/golinters/importas/testdata/importas_several_empty_aliases.yml

```diff
@@ -0,0 +1,9 @@
+linters-settings:
+  importas:
+    alias:
```

Comment:
I'd maybe add `no-unaliased: true` to make sure the rule actually works.

Reply:
I prefer to keep the tests focused on one element because it's easier for maintenance.